tests/test_annotator.py

```diff
@@ -47,10 +57,23 @@ def setUp(self):
             "hue": self.df['color'],
             "order": ["a", "b"],
             "hue_order": ['red', 'blue']}
+        self.order_float_test: list[float] = [1.01, 1.02, 1.03]
 
     def test_init_simple(self):
         self.annot = Annotator(self.ax, [(0, 1)], data=self.data)
 
+    def test_init_float(self):
+        df_float_labels: pd.DataFrame = pd.DataFrame(data=self.data3)
+        ax_float = sns.boxplot(data=df_float_labels, x="x_axis", y="y_axis", order=self.order_float_test)
+        self.annot_float = Annotator(
+            ax_float,
+            pairs=[(1.01, 1.02), (1.02, 1.03)],
+            data=df_float_labels,
+            x="x_axis",
+            y="y_axis",
+            order=self.order_float_test
```

Comment:
Could you make that as a `self.params_float` `dict` in `setUp` as we did for the others?

Reply:
Will do once I solve my `hue` problem.

Reply:
Changes were committed